.ci/update-github-io.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: update-github-io.sh <version>"
+  exit 1
+fi
+
+TARGET_VERSION=$1
+echo TARGET_VERSION="$TARGET_VERSION"
+
+echo "Go to folder where site was build and sources are already at required tag"
+cd target/checkout
+
+echo "Generating web site"
+mvn -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
+
+echo "Come back repo folder"
+cd ../../
+
+echo "Switch to checkstyle.github.io repository"
+if [ -d "../checkstyle.github.io" ] ; then
+  cd ../checkstyle.github.io
```

Comment:
Remove this, it is required only for release on local to reuse resources.
Action has no pre-cloned repos, all should be done from scratch

Reply:
Changes Done